_posts/en/newsletters/2022-02-09-newsletter.md

```diff
@@ -0,0 +1,157 @@
+and vulnerabilities, but it is expected to be minimal since these APIs change
+very infrequently."
+  a1link="https://bitcoincore.reviews/libsecp256k1-748#l-120"
+
+  q2="Can you follow the examples added in the PR? Is anything missing from them?"
+  a2="Participants discussed their experience compiling and running the
+examples, using debuggers, comparing the example code with Bitcoin Core usage,
+and considering the UX for non-Bitcoin users.
+One participant pointed out that not verifying the Schnorr signature
+after producing it was a deviation from the Bitcoin Core code and BIP340
+recommendation. Another participant suggested demonstrating the usage of
+`secp256k1_sha256` before `secp256k1_ecdsa_sign`, as forgetting to
+hash the message could be a potential user footgun."
+  a2link="https://bitcoincore.reviews/libsecp256k1-748#l-193"
+
+  q3="What can happen if a user forgets to do something like verify the
+signature after signing, call `seckey_verify`, or randomize the context?"
+  a3="In the worst case scenario, if there is a flaw in the implementation,
+forgetting to verify the signature after signing could mean accidentally giving
+out an invalid signature. Forgetting to call `seckey_verify` after randomly
+generating a key means there is a (negligible) probability of having an invalid
+key. Randomizing the context is intended to protect against side channel
+attacks; it blinds the intermediary values which have no impact on the end
+result but may be exploited to gain information about the operations performed."
+  a3link="https://bitcoincore.reviews/libsecp256k1-748#l-226"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.2-beta.rc1][lnd 0.14.2-beta] is the release candidate for a
+  maintenance version that includes several bug fixes and a few minor
+  improvements.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23508][] Add getdeploymentinfo RPC FIXME:Xekyo
+
+- [Bitcoin Core #21851][] release: support cross-compiling for arm64-apple-darwin FIXME:adamjonas
+
+- [Bitcoin Core #16795][] updates the `getrawtransaction`, `gettxout`,
+  `decoderawtransaction`, and `decodescript` RPCs to return the inferred
+  [output script descriptor][topic descriptors] for any scriptPubKeys
+  that are decoded.
+
+- [LND #6626][] sets 5% as the default fee for payments routed through
```

Comment:
I believe this should be https://github.com/lightningnetwork/lnd/pull/6226 instead. Pushing a small fix for this.